dev/custom-pages/athletics.scss

```diff
@@ -0,0 +1,643 @@
+
+      li {
+        margin-bottom: .6em;
+      }
+
+      a {
+        font-weight: normal;
+        text-decoration: none;
+
+        &:hover,
+        &:active,
+        &:focus {
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+
+
+  .section-fightsong {
+    @media (min-width: $screen-sm-min) {
+      overflow-x: hidden;
+      position: relative;
+    }
+
+    @media (min-width: $screen-md-min) {
+      min-height: 700px;
+    }
+
+    .section-bgwrap {
+      @media (min-width: $screen-sm-min) {
+        background-image: url($fightsong-bg-sm-url);
+        background-position: right bottom;
+        background-repeat: no-repeat;
+        background-size: 250px auto;
+        bottom: 0;
+        left: 50%;
+        position: absolute;
+        top: 0;
+        transform: translateX(-50%);
+        width: 800px;
+        z-index: -1;
+      }
+
+      @media (min-width: $screen-md-min) {
+        background-size: auto;
+        width: 1130px;
+      }
+    }
+  }
+
+  .fightsong-wrap {
+    @include clearfix();
+  }
+
+  .fightsong-line {
+    @include knockout();
+    border: 3px solid $ucf-gold;
+    clear: both;
+    display: block;
+    font-size: 5vw;
+    letter-spacing: 1px;
+    line-height: 1;
+    padding: .4em .8em;
+    text-transform: uppercase;
+
+    @media (min-width: $screen-sm-min) {
+      font-size: 26px;
+    }
+
+    @media (min-width: $screen-md-min) {
+      font-size: 38px;
+    }
+
+    &:not(:last-child) {
+      margin-bottom: -3px;
+    }
+
+    &:nth-child(odd) {
+      float: left;
+
+      &:nth-child(3n+3) {
+        margin-left: 1em;
+      }
+    }
+
+    &:nth-child(even) {
+      float: right;
+
+      &:nth-child(4n+4) {
+        margin-right: 1em;
+      }
+
+      &:nth-child(6n+6) {
+        margin-right: 1.5em;
+      }
+    }
+  }
+
+
+  .section-events {
+    .section-heading {
+      margin-top: 0;
+    }
+  }
+
+  .callout-events {
+    @media (min-width: $screen-md-min) {
+      background-image: url('#{$events-bg-md-url}');
+      background-position: center right;
+      background-repeat: no-repeat;
+      background-size: auto 100%;
+    }
+
+    .list-chevrons,
+    .events-list {
+      letter-spacing: .04em;
+    }
+
+    .heading-heavy {
+      margin: 2.5em 0 1em;
+    }
+
+    .btn-ucf-gold {
+      border-bottom: 1px solid $ucf-gold;
+      color: $black;
+      font-size: inherit;
+      text-decoration: none;
+
+      &:hover,
+      &:active,
+      &:focus {
+        border-bottom: 1px solid $ucf-metal-gold;
+      }
+    }
+  }
+
+  .events-list {
+    list-style: none;
+    padding-left: 0;
+
+    .events-list-item {
+      @media (min-width: $screen-sm-min) {
+        margin-bottom: 25px;
+        margin-top: 25px;
+      }
+    }
+
+    .event-link {
+      text-decoration: none;
+
+      &:hover,
+      &:active,
+      &:focus {
+        text-decoration: none;
+
+        .event-date,
+        .event-title {
+          text-decoration: underline;
```

Comment:
We should probably keep the hover state on this inline with what's on the home page.